feat: support downloading artifacts in the new emui

Dartoxian opened the pull request:
## Description:
This PR adds support for downloading artifacts as TGZ's from the new emui.

Additionally this PR:
* updates the copy on the files table to File Artifacts
* Makes files non clickable
* Fixing the scrolling behaviour over monaco editors.

## Is this change user facing?
YES

Dartoxian committed:
Fix scrolling over the code editor

Dartoxian committed:
Support artifact download

Dartoxian committed:
Default sort files by filename

Dartoxian committed:
Update with post standup feedback

tedim52 commented on enclave-manager/web/src/components/enclaves/widgets/DownloadFileButton.tsx:
```diff
@@ -0,0 +1,45 @@
+
+  const handleDownloadClick = async () => {
+    setIsLoading(true);
+    // todo: get tgz download instead
```
would it be here Ben?

Dartoxian replied:
yah, needs to be wired through from `KurtosisClient`

Dartoxian committed:
fix: enclave ui feedback implementation (#1725)

## Description:
This PR implements all burdown feedback for:
* Enclave list view (make clickable cells work properly, make headers
highlightable when sortable, image names clickable, make non clickable
styles work with correct styling)
* Enclave overview (remove source tab, fix cell interactivity in table
for ports, merge public private ports in popover, drop ports timeout)
* Enclave logs (link to existing issue rather than suggesting make a new
issue)
* Service view (fix monaco scrolling, remove delete/edit buttons, merge
public/private ports)
* Enclave configuration (copy updates, but not json field update).

## Is this change user facing?
YES

## References (if applicable):
*
https://www.notion.so/kurtosistech/FE-Devconnect-Work-breakdown-af548547e0ec4a4996f75205a4fcbd60#b9f4e5a71aad4347827473e72bc888b2

adschwartz committed:
Merge branch 'main' into feat/download-files

Commit:
new build
